docs/publishing-binaries.md

```diff
@@ -157,10 +157,25 @@ index e00b7b5..22f7cd9 100644
 
 3. Publishing C++ binaries by pushing a commit with `[publish binary]` per https://github.com/mapbox/node-cpp-skel/blob/master/docs/publishing-binaries.md#7-all-done
 
-4. Require your module in downstream applications like:
+
+4. 
+
+**Option A:** Require your module in downstream applications like:
 
 ```js
 "your-module": "https://github.com/<your-org>/<your-module>/tarball/<your-branch>",
 ```
 
 If you're publishing from a private repo, generate a dev release and then reference the url in the appropriate `package.json` file. For example, `zip` the repo, put to S3, and then reference the S3 url in `package.json`. 
+
+**Option B:** Issue a npm dev release after the binary is published. 
+
+Run `npm publish --tag dev` and then require your module in downstream applications like: 
+
+```js
+"your-module": "0.1.0-alpha1",
+```
+
+For npm dev releases, it’s good to use the `--tag <something>` to avoid publishing to the latest tag. If you run `npm publish` then `0.1.0-alpha` is what anyone running `npm install vtquery --save` will receive. 
```

Comment:
I'm struggling to understand the relationship between
```
--tag <something>
```
and the following examples provided:
- `npm publish --tag dev`
-  `0.1.0-alpha`
- `"0.1.0-alpha1",`

Reply:
@GretaCB that is confusing - the example package name should just be `0.1.0-alpha` - `alpha1` was a typo. I've updated this!